frontend/packages/dev-console/src/components/topology/components/nodes/MonitoringDecorator.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Tooltip, TooltipPosition } from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { global_warning_color_100 as warningColor } from '@patternfly/react-tokens';
+import * as UIActions from '@console/internal/actions/ui';
+import { Node } from '@console/topology';
+import Decorator from './Decorator';
+
+interface MonitoringDecoratorProps {
+  radius: number;
+  x: number;
+  y: number;
+  element: Node;
+  onClickTab?: (name: string) => void;
```

Comment:
Rename to just `onClick` with no name param. The name is a const, use it in the connect down below.